services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -302,4 +311,25 @@ public final Device setStatus(final DeviceStatus status) {
     public final DeviceStatus getStatus() {
         return status;
     }
+
+    /**
+     * Sets the authorities granted to this device.
+     *
+     * @param authorities The devices' authorities.
```

Comment:
"device's" - same in getter below.

Reply:
I think this is correct as the this is a possessive of a plural noun:

    The possessive of a plural noun is formed by adding only an apostrophe when the noun ends in s, and by adding both        an apostrophe and s when it ends in a letter other than s.

See: https://www.thepunctuationguide.com/apostrophe.html#possessives

Reply:
But it's about the authorities of *one* device ("this") here, isn't it?